app/src/main/java/com/ruesga/rview/SearchActivity.java

```diff
@@ -657,8 +658,8 @@ private void requestCustomSuggestions(String filter) {
     private CharSequence performFilterHighlight(Suggestion suggestion) {
         Spannable span = Spannable.Factory.getInstance().newSpannable(suggestion.mSuggestionText);
         int pos = 0;
-        final String suggestionNoCase = suggestion.mSuggestionText.toLowerCase();
-        final String filterNoCase = suggestion.mFilter.toLowerCase();
+        final String suggestionNoCase = suggestion.mSuggestionText.toLowerCase(Locale.US);
+        final String filterNoCase = suggestion.mFilter.toLowerCase(Locale.US);
```

Comment:
This two strings came from user input, so Locale.US could not be the right locale, specially when user is searching for user accounts. I suggest to use the current locale from Configuration.locale, and concretely use AndroidHelper.getCurrentLocale() to deal with the new multilocale methods.